assignment2/src/components/Question1.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react'
+
+function Question1() {
+      const [email, setEmail] = useState("")
```

Comment:
Please consider single object like {email, password}  for setting email and password into state.